### DISCUSSION TITLE
“Hey, you need to upgrade soon” Alerts

### DISCUSSION THREAD

**Discussion by psteinroe** [supabase/supabase#9232] [Feature Requests]
Hey,

it would be great if supabase could send their users a warning when they are close to hitting the resource limits on a project. We already have had a downtime because we simply did not look into the utilisation. Ofc, this can be manually implemented using the provided endpoints, but an out-of-the-box alert would be awesome!

**Comment by thorwebdev** [comment]
This is a great idea, thanks! I've flagged with the team and will have them follow up.
